web/packages/teleport/src/services/audit/makeEvent.ts

```diff
@@ -2029,6 +2029,20 @@ export const formatters: Formatters = {
       return `Stable UNIX user for username [${username}] was created`;
     },
   },
+  [eventCodes.AWS_IC_RESOURCE_SYNC_SUCCESS]: {
+    type: 'aws_identity_center.resource_sync.success',
+    desc: 'AWS IAM Identity Center Resource Sync',
+    format: ({ message }) => {
+      return message;
```

Comment:
nit: 

Printing message will make the UI vague: 

![Screenshot 2025-02-12 at 20 45 36](https://github.com/user-attachments/assets/cf928f81-d51a-4688-ad79-e804c52099bc)


The User Audit Event description should be meaningful to the user. I suggest extracting relevant data from the audit event and displaying something like:

AWS IC synchronization successfully completed [groups: X, permission sets: X, account: X]

This way, the user can immediately understand the details of the audit event.

Reply:
Done https://github.com/gravitational/teleport/pull/52059/commits/53881d50a53f6e737578b26633feb193b9fcc27c#diff-5018bbae0e65b4c5bddd40edf2b82046daeac34a2872f97b9c4e0d58d183681fR2045